clientSide/src/components/Product/Product.js

```diff
@@ -0,0 +1,33 @@
+import './Product.css';
+
+function Products(){
+    return(
+
+        <div className="color">
+            <div className="categories">
```

Comment:
Please add alternate text for image tags.

Reply:
I added alternate text for image tage please check it once